bot/bot.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+from sys import stderr
+from traceback import print_exc
+
+
+from bot import constants
+from discord.ext import commands
+
+
+bot = commands.Bot(command_prefix=commands.when_mentioned_or('!'))
+
+if __name__ == '__main__':
+    cogs = [x.stem for x in Path('cogs').glob('*.py')]
```

Comment:
this line could do with a comment to clarify what it's doing. I also don't love it when variables are named `x`.

Reply:
i 100% agree with you. that is now taken care of.